src/runtime/config/configuration-qemu.toml.in

```diff
@@ -352,6 +352,11 @@ pflashes = []
 # Default false
 #hotplug_vfio_on_root_bus = true
 
+# In a confidential compute environment hot-plugging can compromise
+# security. Enable cold-plugging of VFIO devices to a root-port.
+# The default setting is  "no-port", which means disabled. 
+#cold_plug_vfio = "root-port" 
```

Comment:
please list `switch-port` and explain the difference w/ `root-port` so that users know which one to set here. From the code below, both of them are acceptable, right?

Reply:
The `switch-port` is coming with the next PR for `hot_plug_vfio=no-port, bridge-port, root-port, switch-port`  The values are just placeholders. I will add the description with the follow-up PR https://github.com/kata-containers/kata-containers/pull/4492

Reply:
So essentially #4495  will resolve both of your comments